app/processing/validate-sequence.js

```diff
@@ -19,6 +19,8 @@ const getSequence = (schemeId, filename) => {
     case bps.schemeId:
     case fdmr.schemeId:
       return Number(filename.substr(5, 4))
+    case es.schemeId:
+      return Number(filename.substr(24, 4))
```

Comment:
should this be `Number(filename.substr(23, 4)`, think you are missing the first digit unless its not needed? Could add a unit test as well. 

I tested on this filename and missed first digit: `GENESISPayReq_20230104_5394.gne`

Reply:
Ooops good spot.  Fixed and tests added.